packages/google_maps_flutter/ios/google_maps_flutter.podspec

```diff
@@ -17,5 +17,6 @@ A new flutter plugin project.
   s.dependency 'Flutter'
   s.dependency 'GoogleMaps'
   s.static_framework = true
+  s.compiler_flags = '-fno-modules'
```

Comment:
What problem does this flag solve? What does it do, and is that in any way related to the use of `s.static_framework = true`? I'm asking because I haven't encountered the build error you refer to, so I'm not sure how/why the flag helps. If you know, please add a clarifying comment in the `podspec`.

Please bump the version number in `pubspec.yaml` and add a `CHANGELOG.md` entry.